README.md

```diff
@@ -65,3 +65,16 @@ Use `py.test` to execute the unit tests.
 # License
 
 GPLv3. See the [license](license.txt) file.
+
+# Network Protocol
```

Comment:
I choice a file into the repo to have a protocol version for each version.
A wiki is central ...
If we have a file we can discuss stuff and people can with PR's update the file.